### ISSUE TITLE
"No valid wallaby.js license found, it started in trial mode." shown at every started although successfully activated

### ISSUE THREAD

**Issue by peteralbert** [wallabyjs/public#1039]
### Issue description or question
I purchased a license and activated it in VS Code. When I start Wallaby, I get the warning `No valid wallaby.js license found, it started in trial mode. If you would like to purchase a license, please visit our website. If you have already purchased the license, please activate it.`

Clicking on Activate License shows the entered license, pressing enter shows `Wallaby.js is licensed to: )쯚F�data2impact #vmkau. Free upgrades until: 22/02/2018` (including the odd unicode!).

Then 9 out of 10 cases, I can see in the Wallaby.js Console that all files are parsed - and then the service stops without running the tests. Here's the log:

```
[Info]​​​​​ Wed, 22 Feb 2017 17:13:18 GMT wallaby:project Wallaby config: c:\Users\peter\...\wallaby.js
​​​​​[Info]​​​​​ Started Wallaby.js Core v1.0.385
​​​​​[Info]​​​​​ Wed, 22 Feb 2017 17:13:19 GMT wallaby:fs File added: src/main.ts
​​​​​[Info]​​​​​ Wed, 22 Feb 2017 17:13:19 GMT wallaby:fs File added: src/polyfills.ts
...
[Info]​​​​​ Wed, 22 Feb 2017 17:13:20 GMT wallaby:fs File added: src/app/content/region/scorecard/period/wave/wave.module.ts
​​​​​[Info]​​​​​ Wed, 22 Feb 2017 17:13:21 GMT wallaby:project File cache: C:\Users\peter\.vscode\extensions\WallabyJs.wallaby-vscode-1.0.45\projects\1f3e8aa258e56ce3
​​​​​[Info]​​​​​ Wed, 22 Feb 2017 17:13:21 GMT wallaby:uiService Listening port 51235
​​​​​[Info]​​​​​ Wallaby.js stopped
```

 In the other case, wallaby then runs normally but after a few minutes informs me that it'll stop soon as it is running in Trial mode.

Next time I start wallaby, I again have to enter the license.

### Code editor or IDE name and version

Visual Studio Code v1.9.1
Wallaby 1.0.45

### OS name and version

Windows 10 64bit

**Comment by peteralbert** [comment]
Hah, I think I resolved the issue: I copied the license key including the `Key #vmkau:` from the email. When I remove this, it seems to activate permanently.

This still seems like an issue to me. Probably best to do either of these options:

* remove the `Key #vmkau:` part from the key in the email
* provide better instructions in the email, which part of the key is to be copied
* or update the license key verification/storing to strip this part away
